documentation/extracts.md

```diff
@@ -2,19 +2,22 @@
 layout: sub-navigation.njk
 order: 12
 title: Extracts
-description: Extracts in CSV file format can be generated to give users the ability to produce reports.
+description: Extracts in CSV file format can be generated.
 ---
 
 [[toc]]
 
-DECS provides a number of extracts in CSV file format to give users the ability to produce reports. To get access to these reporting extracts, a specific role has to be added to a user within keycloak.
+DECS provides a number of extracts in CSV file format. To get access to these extracts, a specific role has to be added to a user within keycloak.
```

Comment:
Wonder if we can point the user to the bottom section within here for the roles?